app/lib/settings/src/subpages/timetable/timetable_settings_page.dart

```diff
@@ -172,9 +172,9 @@ class _ICalLinks extends StatelessWidget {
         .read<SubscriptionService>()
         .hasFeatureUnlocked(SharezonePlusFeature.iCalLinks);
     return ListTile(
-      title: const Text("Stundenplan exportieren (iCal)"),
+      title: const Text("Termine, Prüfungen, Stundenplan exportieren (iCal)"),
       subtitle: const Text(
-        "Stundenplan in Google Kalender, Apple Kalender, etc. einbinden",
+        "Stundenplan oder Termine/Prüfungen in Google Kalender, Apple Kalender, etc. einbinden",
```

Comment:
```suggestion
        "Synchronisierung mit Google Kalender, Apple Kalender usw.",
```

Maybe something like this. Because otherwise it becomes it a bit too long and half of it would be duplication.